cpp/include/raft/linalg/qr.cuh

```diff
@@ -70,6 +69,51 @@ void qrGetQR(const raft::handle_t& handle,
 {
   detail::qrGetQR(handle, M, Q, R, n_rows, n_cols, stream);
 }
+
+/**
+ * @brief compute QR decomp and return only Q matrix
+ * @param handle raft::handle_t
+ * @param M Input raft::device_matrix_view
+ * @param Q Output raft::device_matrix_view
+ */
+template <typename ElementType, typename IndexType = std::uint32_t>
```

Comment:
This file following the snake-cased format that we've been using everywhere else: (e.g. `qr_get_q`)